### DISCUSSION TITLE
Making some rows unselectable in the bulk mode

### DISCUSSION THREAD

**Discussion by rpringadi** [rappasoft/laravel-livewire-tables#1243] [General]
First of all thank you for this wonderful table component.

I managed to load up users into the table. 
I can perform bulk action using the bulit-in check box column. 
My bulk action would be to delete users. 
Works fine, however some users should not be delete-able. Thus, these users should not be selectable either by putting a check box manually on a single row, or by select all.  

Is there a way to make a certain row to be unselectable? Thanks!

**Comment by lrljoe** [comment]
That is a very good question!

While you can certainly stop the deletion on the backend, there's nothing in Bulk Actions as it stands to disable the checkbox.

It's a good request, so I'll convert it into a feature request. 
 I can't promise it will be done very soon, as there are a couple of requests that I've been meaning to get to for far too long, but the idea is a solid one so it shouldn't be too long.

You could do it with some custom code in the meantime, let me know if you need pointers for this.

The issue here is, you're selecting a checkbox before a bulk action.

In your case, it's a delete. 

But you may have a delete bulk action, an export bulk action, and an "activate" bulk action, so just disabling the checkbox may not be desirable!

What I'll probably end up doing is adding an option for a custom blade for the bulk action checkbox.

**Comment by rpringadi** [comment]
Thanks @lrljoe 

> You could do it with some custom code in the meantime, let me know if you need pointers for this.

Yes please, I would like to know / learn more about this component.
What alternative can I do to make a row to be unselectable from the bulk checkbox?

**Reply by lrljoe** [reply]
If you publish the bulk action checkboxes view. Then you can wrap it in an @if in the blade file
